api/validation/schemas.py

```diff
@@ -53,72 +51,72 @@ class GetInstanceTypesSchema(Schema):
 
 
 class GetDcvSessionSchema(Schema):
-    user = fields.String()
-    instance_id = fields.String(required=True)
+    user = fields.String(validate=validate.Length(max=64))
+    instance_id = fields.String(required=True, validate=validate.Length(max=60))
     region = fields.String(validate=aws_region_validator)
 
 GetDcvSession = GetDcvSessionSchema(unknown=INCLUDE)
 
 
 class QueueStatusSchema(Schema):
-    user = fields.String()
-    instance_id = fields.String(required=True)
+    user = fields.String(validate=validate.Length(max=64))
+    instance_id = fields.String(required=True, validate=validate.Length(max=60))
     region = fields.String(required=True, validate=aws_region_validator)
 
 QueueStatus = QueueStatusSchema(unknown=INCLUDE)
 
 
 class ScontrolJobSchema(Schema):
-    user = fields.String()
-    instance_id = fields.String(required=True)
-    job_id = fields.String(required=True)
+    user = fields.String(validate=validate.Length(max=64))
+    instance_id = fields.String(required=True, validate=validate.Length(max=60))
+    job_id = fields.String(required=True, validate=validate.Length(max=256))
     region = fields.String(required=True, validate=aws_region_validator)
 
 ScontrolJob = ScontrolJobSchema(unknown=INCLUDE)
 
 
 class CancelJobSchema(Schema):
-    user = fields.String()
-    instance_id = fields.String(required=True)
-    job_id = fields.String(required=True)
+    user = fields.String(validate=validate.Length(max=64))
+    instance_id = fields.String(required=True, validate=validate.Length(max=60))
+    job_id = fields.String(required=True, validate=validate.Length(max=256))
     region = fields.String(required=True, validate=aws_region_validator)
 
 CancelJob = CancelJobSchema(unknown=INCLUDE)
 
 
 class SacctSchema(Schema):
-    user = fields.String()
-    instance_id = fields.String(required=True)
-    cluster_name = fields.String(required=True, validate=is_alphanumeric_with_hyphen)
+    user = fields.String(validate=validate.Length(max=64))
+    instance_id = fields.String(required=True, validate=validate.Length(max=32))
```

Comment:
question: is the instance_id for DCV referring to a different object? If not, why in one case I have 60 and in this case 32?

Reply:
My bad, I'll correct it and re-push! Thanks